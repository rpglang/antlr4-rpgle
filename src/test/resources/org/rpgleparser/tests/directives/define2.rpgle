      /if defined (header_INS002)                                                                                     
      /undefine header_INS002                                                                                         
      /endif