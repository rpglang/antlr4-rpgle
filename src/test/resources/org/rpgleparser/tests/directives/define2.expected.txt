/*===TOKENS===*/
HIDDEN:DIRECTIVE         < />
HIDDEN:DIR_IF            <if>
HIDDEN:DIR_DEFINED       <defined>
HIDDEN:'('               <(>
HIDDEN:')'               <)>
HIDDEN:DIRECTIVE         < />
HIDDEN:DIR_UNDEFINE      <undefine>
HIDDEN:DIR_OtherText     <header_INS002>
HIDDEN:EOL               <>
HIDDEN:DIRECTIVE         < />
HIDDEN:DIR_ENDIF         <endif>
/*===TREE===*/
r
/*======*/